packages/paypal-js/src/load-script.ts

```diff
@@ -64,6 +64,10 @@ export function loadCustomScript(
 
     const { url, attributes } = options;
 
+    if (attributes && !attributes["data-js-sdk-library"]) {
```

Comment:
`loadCustomScript` will always have `attributes` when called by `paypal-js` -> `loadScript`, but checking for `attributes` here, since `react-paypal-js` calls `loadCustomScript` without `attributes` in `getBraintreeNamespace` [here](https://github.com/paypal/react-paypal-js/blob/e365997c6c31bf459438f2b191884a1507b20b97/src/components/braintree/utils.ts#L96-L97).